Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -8191,6 +8191,20 @@ WebAnimationsCustomFrameRateEnabled:
     WebCore:
       default: false
 
+WebAnimationsOverallProgressPropertyEnabled:
+  type: bool
+  status: testable
+  category: animation
+  humanReadableName: "Web Animations 'overallProgress' property"
+  humanReadableDescription: "Support for the 'Animation.overallProgress' property"
```

Comment:
This syntax is misleading. It's not a static property after all. I usually pick one of:

* Animation's overallProgress
* Animation.prototype.overallProgress

Reply:
```suggestion
  humanReadableDescription: "Support for Animation's overallProgress property"
```